### ISSUE TITLE
Hints for installation dirs

### ISSUE THREAD

**Issue by FabianInostroza** [joefitzgerald/go-config#26]
The package description says: "You can optionally configure the package to provide hints for go installations tools." Where can I add those hints? I couldn't find where

**Comment by joefitzgerald** [comment]
Good point. The ability to do this never made it into 1.0. I will update the readme.
